src/task_monitor/tasks/create_batches.rs

```diff
@@ -39,7 +39,7 @@ pub async fn create_batches(
 
     // We start a timer and force it to perform one initial tick to avoid an
     // immediate trigger.
-    let mut timer = time::interval(app.config.app.batch_insertion_timeout);
+    let mut timer = time::interval(Duration::from_secs(5));
```

Comment:
Why change from a config value to a constant one?

Reply:
We use 5 seconds constant in other tasks and insertion timeout is very big. And when restarted it was waiting batch_insertion timeout before doing a batch while there was enough identities already waiting. :)